tests/test_Model.py

```diff
@@ -315,12 +317,12 @@ def test_model_solve_display_argument(ok_small, capsys):
 
     # First solve with display turned off. We should not see any output in this
     # case.
-    model.solve(stop=MaxIterations(1000), seed=0, display=False)
+    model.solve(stop=MaxIterations(10), seed=0, display=False)
     printed = capsys.readouterr().out
     assert_equal(printed, "")
 
     # Now solve with display turned on. We should see output now.
-    res = model.solve(stop=MaxIterations(1000), seed=0, display=True)
+    res = model.solve(stop=MaxIterations(10), seed=0, display=True)
     printed = capsys.readouterr().out
```

Comment:
1000 iters on debug builds takes a few seconds. This single tests adds like 25% time to a full test run, which seems a bit excessive :-).